unofficial/c511027665.lua

```diff
@@ -0,0 +1,40 @@
+--Abyss Performance
+--Made by Beetron-1 Beetletop
+local s,id=GetID()
+function s.initial_effect(c)
+	--chain attack
+	local e1=Effect.CreateEffect(c)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCode(EVENT_BATTLE_DESTROYING)
+	e1:SetCondition(s.condition)
+	e1:SetTarget(s.target)
+	e1:SetOperation(s.activate)
+	c:RegisterEffect(e1)
+end
+function s.condition(e,tp,eg,ep,ev,re,r,rp)
+	local tc=eg:GetFirst()
+	local bc=tc:GetBattleTarget()
+	return tc:IsRelateToBattle() and tc:IsStatus(STATUS_OPPO_BATTLE) and tc:IsControler(tp) and tc:IsSetCard(0x10ec)

```

Comment:
Use archetype constants (`SET_ABYSS_ACTOR`), also missing `listed_series` below the initial effect.